config: fix SSL detection

The @WEBSOCK_HAVE_SSL@ is replaced at configure time with either a 1 or 0.
The rest of the code is simply checking to see if WEBSOCK_HAVE_SSL is
defined at all. Using a #undef WEBSOCK_HAVE_SSL will cause configure
to either comment it out or change the line to
"#define WEBSOCK_HAVE_SSL 1".

Signed-off-by: Clayton Shotwell <clayton.shotwell@rockwellcollins.com>

diff -durN a/src/websock_config.h.in b/src/websock_config.h.in
--- a/src/websock_config.h.in
+++ b/src/websock_config.h.in
@@ -4,6 +4,6 @@
 #define WEBSOCK_PACKAGE_STRING @WEBSOCK_PACKAGE_STRING@
 #define WEBSOCK_PACKAGE_VERSION @WEBSOCK_PACKAGE_VERSION@
 #define WEBSOCK_PACKAGE_NAME @WEBSOCK_PACKAGE_NAME@
-#define WEBSOCK_HAVE_SSL @WEBSOCK_HAVE_SSL@
+#undef WEBSOCK_HAVE_SSL
 
 #endif
